src/scss/components/optional/_loading-indicator.scss

```diff
@@ -33,16 +41,17 @@ $loading-indicator-spacing: 20px;
 
   .c-spinner {
     margin-right: $loading-indicator-spacing;
-    border: $loading-indicator-borderSize solid $loading-indicator-color;
-    border-top: $loading-indicator-borderSize solid
-      $loading-indicator-borderColorOpaque;
     border-radius: 50%;
     animation: spin 1s linear 0s infinite;
 
-    @if $loading-indicator-size == 'large' {
+    @include spinnerColor();
+
+    @if $loading-indicator-size=='large' {
       width: $loading-indicator-size-large;
       height: $loading-indicator-size-large;
-    } @else {
+    }
+
+    @else {
```

Comment:
These changes are being done by my VS Code, following our `eslint` file, I believe. Is this expected? (I haven't worked on `fozzie` for a while now).